SPECS/safeclib/safeclib.spec

```
@@ -0,0 +1,72 @@
+Summary:        C11 Annex K functions
+Name:           libsafec
+Version:        3.7.1
+Release:        1%{?dist}
+License:        MIT
+URL:            https://github.com/rurban/safeclib
+Source0:        https://github.com/rurban/safeclib/releases/download/v3.7.1/safeclib-%{version}.tar.xz
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  gcc
+BuildRequires:  make
+%if %{with_check}
+BuildRequires:  tzdata
+%endif
+
+%description
+Extension library providing C11 Annex K functions
+
+%package -n libsafec-devel
+Summary:        Development packages for libsafec
+Requires:       libsafec = %{version}-%{release}
+
+%description -n libsafec-devel
+Development files for libsafec
+
+%package -n libsafec-check
+Summary:        Tools to detect use of unsafe libc APIs
+
+%description -n libsafec-check
+Tools to detect use of unsafe libc APIs
+
+%prep
+%setup -q -n safeclib-%{version}
+
+%build
+autoreconf -Wall --install
+%configure --disable-static --enable-debug --enable-strmax=0x8000
+%make_build
+
+%if %{with_check}
+%check
+export TZ=Europe/London
+make check %{?_smp_mflags}
+%endif
```

No need to cut out the `%check` section for non-test builds. This section is automatically skipped anyway.
```suggestion
%check
export TZ=Europe/London
make check %{?_smp_mflags}
```